panther_driver/src/PantherKinematics.py

```diff
@@ -17,6 +17,7 @@ def __init__(self):
         self.robot_length = 0
         self.wheel_radius = 0  # Distance of the wheel center, to the roller center
         self.encoder_resolution = 0
+        self.gear_ratio = 0
```

Comment:
General comment regarding this module. Please remove unused imports.

Reply:
Done ✓